pkg/models/email_address.go

```diff
@@ -0,0 +1,14 @@
+package models
+
+// EmailAddress represents an email address
+type EmailAddress string
+
+// String returns the email address as a string
+func (e EmailAddress) String() string {
```

Comment:
It might be a personal thing, but I don't like using raw casts because it bakes into consuming code that an `EmailAddress` is really just a string underneath. Having a constructor and an explicit `String` method means that we can do things like change the implementation of the type to be a struct in the future without breaking the code that uses the type.

The constructor also provides a place to do format validation like @sarboc is suggesting, although that would require changing the signature of the constructor to also return an `error` for when the validation fails.